doajtest/unit/test_article_acceptable_and_permissions.py

```diff
@@ -42,13 +42,24 @@ def test_is_acceptable(self, value, kwargs):
         ft = "https://example.com" if ft_arg == "exists" else None
         incorrect_issn_count = True if kwargs.get("incorrect_issn_count") == "yes" else False
         pissn_eissn_match = True if kwargs.get("pissn_eissn_match") == "yes" else False
+        journal_in_doaj = True if kwargs.get("journal_in_doaj") == "yes" else False
+
+        journal_source = JournalFixtureFactory.make_journal_source(in_doaj=False)
+        journal = Journal(**journal_source)
+        jbib = journal.bibjson()
+        jbib.add_identifier(jbib.P_ISSN, '0000-0000')
+        jbib.add_identifier(jbib.E_ISSN, '1111-1111')
+        if journal_in_doaj:
+            journal.set_in_doaj(True)
+        journal.save(blocking=True)
 
         if pissn_eissn_match:
             article_source = ArticleFixtureFactory.make_article_source(pissn="0000-0000", eissn="0000-0000")
         else:
             article_source = ArticleFixtureFactory.make_article_source(pissn="0000-0000", eissn="1111-1111")
 
         article = Article(**article_source)
+        article.add_journal_metadata()
```

Comment:
I'm slightly unsure that we should be doing this here.  Adding the journal metadata is something that the `create_article` method does after the `is_acceptable` check, so in general this function wouldn't have run before the article is passed to `is_acceptable`.  Doing it here might not give us a totally legitimate test.  What's the reason for adding it, does it provide something to the article that's required for it to process?

Reply:
No, it is not needed. Tests pass without it. I suspect it is a leftover from some previous approach o.O I deleted that.